service/templates/patch_version.html

```diff
@@ -22,8 +22,7 @@
 
 {{define "content"}}
 <div ng-controller="PatchController">
-
-  <div ng-show="patch.Alias == __commit_queue">
+  <div ng-show="patch.Alias == '__commit_queue'">
```

Comment:
It doesn't matter here, but in general you should use === and !==. Otherwise javascript will do a type conversion, which can return unexpected results if comparing to 0 or ""